components/Navigator/components/Content/components/Search/index.jsx

```diff
@@ -175,13 +205,13 @@ class Search extends Component {
             ref={(element) => { this.inputElement = element; }}
             onFocus={this.handleFocus}
           />
-          <div
-            className={styles.overlay}
-            onClick={this.handleOverlayClick}
-            role="button"
-            aria-hidden
-          />
         </form>
+        <div
```

Comment:
we already have a Backdrop component (@shopgate/pwa-common/components/Backdrop) which is uses for the navdrawer and sheet for example. Would be nice to use the same component here